low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -306,12 +332,24 @@ class SSPLDevDeploy:
         # Execute setup commands
         for cmd in commands:
             print("INFO: Executing command: '%s'" % (cmd))
-            self._execute_cmd(cmd)
+            out, rc = self._execute_cmd(cmd)
+            if rc:
+                raise DeployError(rc, "%s - CMD: %s", out, cmd)
 
     def _execute_cmd(self, cmd):
         """Executing commands."""
-        returncode = subprocess.call(cmd.split(), shell=False)
-        return returncode
+        _out = None
+        process = subprocess.Popen(cmd.split(),
```

Comment:
Use simpleProcess if it can serve the purpose, otherwise its fine for dev deploy script

Reply:
We start using `subprocess` even for installing cortx-py-utils package. SimpleProcess will be available only after that package installation. So we stay with this one.

Reply:
>We start using subprocess even for installing cortx-py-utils package.

This is understood right.

> SimpleProcess will be available only after that package installation. So we stay with this one.

Ofcourse, i meant to install cortx-utils can be installed and do related imports only in specific functions, which can be invoked only after cortx-utils installation.
But dev-deploy am fine with this too, as it would be used in dev/test env alone & would be used for sometime only.